pkg/sql/database.go

```diff
@@ -55,6 +54,15 @@ func open(datasource string) (*DB, error) {
 	return db, err
 }
 
+// SplitDataSource splits the datasource into drivername and datasource name
+func SplitDataSource(datasource string) (string, string, error) {
+	dses := strings.Split(datasource, "://")
+	if len(dses) != 2 {
+		return "", "", fmt.Errorf("Expecting but cannot find :// in datasource %v", datasource)
```

Comment:
Sometimes, if `datasource` is an empty string, this error message is not showing the actual datasource, a little confusing.